gatsby_fp/src/components/Layout/Body/Fields/IntellCont/SubHeading.js

```diff
@@ -87,6 +87,9 @@ export default function IntellCont({ intellContList, publicationType }) {
       if (element.title_secondary) {
         liString += `. ${element.title_secondary}`
       }
+      if (element.journal_name) {
+        liString += `. <i>${element.journal_name}</i>`
```

Comment:
Using `i` instead of `em` here is intentional. Sources:

- https://accessibility.psu.edu/boldfacehtml/ (compare examples of "semantic uses of bold & italics" with "visual uses of bold & italics")
- https://blog.logrocket.com/youre-using-em-wrong/#when-not-to-use-em
- https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element

Reply:
https://support.siteimprove.com/hc/en-gb/articles/115002726312-Accessibility-Bold-vs-Strong-and-Italic-vs-Emphasis says "<b> and <i> should not be used to style text when the author wants to emphasize a word or passage of text".

Reply:
According to https://support.siteimprove.com/hc/en-gb/articles/115002726312-Accessibility-Bold-vs-Strong-and-Italic-vs-Emphasis:
`<b>` and `<i>` should not be used to style text when the author wants to emphasize a word or passage of text

Reply:
Not sure which source to trust, I don't mind approving this so long as you're 100% sure that it does not violate the accessibility standards.

Reply:
That siteimprove article is actually not a counterpoint to what I'm saying. The article, and the piece you quoted, are explaining how `em` should be used instead of `i` when the semantic meaning is "emphasis" (a.k.a. "stress," referring to the way a speaker would raise their voice). Nobody in their right mind is raising their voice when reading a citation. Citation style involves the use of italics for the purpose of offsetting the title of the work, not for the purpose of emphasis.

Reply:
Wikipedia also follows what I'm saying, and they have very good style guides. For some examples, see https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Titles_of_works (inspect the italicized text to see how they use `<i>` in this case). Some more clarification is found at https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Text_formatting#Italic_face which explains:

> Other, non-emphasis, uses of italics on Wikipedia should use ''...'' markup, not <em> or {{em}} markup.

(In wikis, `''...''` renders as `<i>...</i>`.)